INSTALL.md

```diff
@@ -19,7 +19,9 @@
 
 Required Software:
 * C/C++ Compiler
-* [DPC++ Compiler](https://www.intel.com/content/www/us/en/developer/tools/oneapi/dpc-compiler.html)
+* [DPC++ Compiler](https://www.intel.com/content/www/us/en/developer/tools/oneapi/dpc-compiler.html) if building with SYCL support
+* Python version 3.7 or higher
```

Comment:
i think 3.7 as well as 3.8 are not officially supported

Reply:
Thanks. Updated it to mention version 3.9 or higher.